Microsoft.Toolkit.Uwp.UI/Extensions/HyperlinkExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Windows.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Documents;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="Hyperlink"/> content element that allows
+    /// it to invoke a <see cref="ICommand"/> when clicked
+    /// </summary>
+    public static class HyperlinkExtensions
+    {
+        private static void OnCommandPropertyChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            Hyperlink hyperlink = sender as Hyperlink;
+
+            if (hyperlink != null)
+            {
+                ICommand command = args.NewValue as ICommand;
+
+                if (command != null)
+                {
+                    hyperlink.Click += OnHyperlinkClicked;
```

Comment:
You need to unsubscribe to this first before subscribing, otherwise you have the potential have multiple event handlers on this. I would suggest putting the unsubscribe line before the if, then just have that one if, with no else.